BuildScript.Nuke/Build.RunTest.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using GlobExpressions;
+using Nuke.Common;
+using Nuke.Common.IO;
+using Nuke.Common.Tooling;
+using Nuke.Common.Tools.Docker;
+using Nuke.Common.Tools.DotNet;
+using Nuke.Common.Tools.MSBuild;
+using Nuke.Common.Utilities.Collections;
+using Remotion.BuildScript.Nuke.GenerateTestMatrix;
+using Serilog;
+
+public partial class Build : NukeBuild, ICleanup
+{
+  private readonly TestArgumentHelper _testArgumentHelper = new();
+
+  [Parameter("Path to MSBuildExtensionPack for running the TestSetupBuildFile")]
+  private string MSBuildExtensionPackPath { get; } = "";
+
+  private Target RunTests => _ => _
+      .DependsOn(CompileTestBuild)
+      .DependsOn<ICleanup>(x => x.CleanFolders)
+      .OnlyWhenStatic(() => !SkipTests)
+      .Executes(() =>
+          {
+            var createTestConfigurations = new CreateTestConfigurations
+            (
+                supportedBrowsers: SupportedBrowsers,
+                supportedPlatforms: SupportedPlatforms,
+                supportedDatabaseSystems: SupportedDatabaseSystems,
+                supportedExecutionRuntimes: SupportedExecutionRuntimes,
+                supportedTargetRuntimes: SupportedTargetRuntimes,
+                enabledBrowsers: Browsers,
+                enabledDatabaseSystems: DatabaseSystems,
+                enabledTargetRuntimes: TargetRuntimes,
+                enabledExecutionRuntimes: ExecutionRuntimes,
+                enabledPlatforms: Platforms,
+                enabledConfigurationIDs: Configuration,
+                testOutputFiles: TestProjectFiles
+            );
+            var output = createTestConfigurations.CreateTestMatrix();
+            if (output.Length == 0)
+              Log.Warning("The created test matrix from the test configuration is empty!");
```

Comment:
Could you just return at this point? There is no benefit in continuing to execute this method if no tests will be executed anyway.